packages/resolver/lib/resolver.ts

```diff
@@ -3,17 +3,26 @@ const expect = require("@truffle/expect");
 const provision = require("@truffle/provisioner");
 
 import { ResolverSource } from "./source";
-import { constructSources } from "./sources";
+import { EthPMv1, NPM, GlobalNPM, FS, Truffle } from "./sources";
 
 export class Resolver {
   options: any;
   sources: ResolverSource[];
 
-  constructor(options: any) {
+  constructor(options: any, includeTruffleSource: boolean) {
     expect.options(options, ["working_directory", "contracts_build_directory"]);
 
     this.options = options;
-    this.sources = constructSources(options);
```

Comment:
I was gonna use that abstraction! One of these days... I swear...

Probably good call removing it.